src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private static List<Fruit> fruitsStore = new ArrayList<>();
```

Comment:
I think its better to store fruits in Map<Fruit, Integer> where we can store fruits and it's quantuty